test/integration/activate_int_test.go

```diff
@@ -335,7 +347,7 @@ func (suite *ActivateIntegrationTestSuite) TestActivate_FromCache() {
 	defer ts.Close()
 
 	cp := ts.SpawnWithOpts(
-		e2e.WithArgs("activate", "ActiveState-CLI/small-python", "--path", ts.Dirs.Work),
+		e2e.WithArgs("activate", "ActiveState-CLI/Python2", "--path", ts.Dirs.Work),
```

Comment:
Why change these? I'm a little concerned we're fixing tests just so they pass without ensuring that it actually fixes the user facing issue.